src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -57,59 +98,69 @@ export default {
 	},
 	data() {
 		return {
+			email: null,
+			isGuestCheckout: true,
 			enableCheckoutButton: false,
 			paymentTypes: ['paypal', 'card', 'applePay', 'googlePay'],
 		};
 	},
+	computed: {
+		// A named slot will be created for each validator that fails, where the name of the slot will
+		// be the key of the validator that failed in the validators object
+		emailErrors() {
+			if(!this.email) return;
+			return getFailures({ email }, this.email);
+		}
+	},
 	methods: {
 		validateBasketAndCheckout() {
 			this.$emit('updating-totals', true);
 			// Validate Basket prior to starting
 			this.validateBasket()
-				.then(validationStatus => {
-					if (validationStatus === true) {
-						// request payment method
-						this.submitDropInPayment();
-					} else {
-						// validation failed
+					.then(validationStatus => {
```

Comment:
please clean up all the extra tabs through these files, there was no code change so these tab indexes shouldn't need to change.

Reply:
@mcstover I agree. Sorry the PR was still in draft mode. Will have it ready in a few minutes for review. Thanks